featuretools_tsfresh_primitives/utils.py

```diff
@@ -0,0 +1,46 @@
+import importlib
+
+import numpy as np
+import pandas as pd
+
+
+def _camel_case(snake_str):
+    words = snake_str.split('_')
+    titled_words = map(str.title, words)
+    return ''.join(titled_words)
+
+
+def _tsfresh_to_featuretools(fc_name):
+    module = importlib.import_module('featuretools_tsfresh_primitives')
```

Comment:
I think we could import the featuretools_tsfresh_primitives module at the top of the file and get the same result.  Would there be a tradeoff to using that instead of using importlib here?

Reply:
I'm not aware of any tradeoff. I implemented your solution and tests are passing.